### PR TITLE
Update pod-lifecycle.md

### PR DESCRIPTION
**Pull request by Rohan-Dah**
Changed the back-off delay time from (10s, 20s, 40s, …) to (0s, 10s, 20s, 40s, ...) on line 155

### CODE REVIEW

File: content/en/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 1]
```diff
@@ -152,8 +152,7 @@ and Never. The default value is Always.
 
 The `restartPolicy` applies to all containers in the Pod. `restartPolicy` only
 refers to restarts of the containers by the kubelet on the same node. After containers
-in a Pod exit, the kubelet restarts them with an exponential back-off delay (10s, 20s,
-40s, …), that is capped at five minutes. Once a container has executed for 10 minutes
+in a Pod exit, the kubelet restarts them with an exponential back-off delay (0s, 10s, 20s, 40s, ...), that is capped at five minutes. Once a container has executed for 10 minutes
```
Comment (windsonsea) [inline content/en/docs/concepts/workloads/pods/pod-lifecycle.md:155]:
```suggestion
in a Pod exit, the kubelet restarts them with an exponential back-off delay (0s, 10s, 20s, 40s, ...),
that is capped at five minutes. Once a container has executed for 10 minutes
```

**Review by windsonsea — COMMENTED** [review]
Using a same style can make it prettier.

**Comment by Ritikaa96** [comment]
Hi @Rohan-Dah Are you able to address the reviews under this PR? Thanks !

**Comment by tengqm** [comment]
/close
due to merge conflict not resolved.
Please reopen and rebase if still interested in this fix.

**Pull request closed without merging**